config/environments/development.rb

```diff
@@ -42,14 +42,11 @@
   # Don't care if the mailer can't send.
   config.action_mailer.raise_delivery_errors = false
   config.action_mailer.perform_caching = false
-
-  if ENV["NOTIFY_API_KEY"].present?
-    config.action_mailer.delivery_method = :notify
-    config.action_mailer.notify_settings = {
-      api_key: ENV.fetch("NOTIFY_API_KEY")
-    }
-  end
-
+ 
+  config.action_mailer.delivery_method = :notify
```

Comment:
how does this relate to the chance?

Reply:
does not, was just in the process of removing it got pushed by accident